### PR TITLE
Searchbar

### PR DESCRIPTION
**Pull request by celinevo**

### CODE REVIEW

File: src/pages/CreepypastaPage.js [hunk 1]
```diff
@@ -38,11 +56,12 @@ const Link = styled(NavLink)`
...
   margin: 0;
-  margin-top: -25px;
+  margin-top: -30px;
 `;
 
 const Grid = styled.main`
   display: grid;
   gap: 20px;
   padding: 20px;
+  padding-top: 0px;
```
Comment (ahohnsen) [inline src/pages/CreepypastaPage.js]: You could combine this two like so:

```suggestion
  padding: 0 20px 20px;
```

File: src/components/Searchbar.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+export default function Searchbar({ handleChange, searchValue }) {
+  return (
+    <SearchForm>
+      <label htmlFor="search-input">
+        <VisuallyHidden>Search Spookie Story</VisuallyHidden>
```
Comment (ahohnsen) [inline src/components/Searchbar.js:7]: Can you tell me why you added this paragraph if it is not visible?

File: src/components/Searchbar.stories.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+export default {
+  title: 'Components/Searchbar',
+  component: Searchbar,
+};
+
+const Template = args => <Searchbar {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
```
Comment (ahohnsen) [inline src/components/Searchbar.stories.js]: If you do not have any arguments, you can skip the template if you like:

```suggestion

export const Default = <Searchbar />
```

File: src/pages/CreepypastaPage.js [hunk 4]
```diff
@@ -1,27 +1,45 @@
...
-            wordcount={creepypasta.wordcount}
-            onBookmarkClick={() => handleBookmarkClick(creepypasta.id)}
-            isBookmarked={creepypasta.isBookmarked}
-          />
-        ))}
+        <Searchbar handleChange={handleChange} searchValue={searchValue} />
+        {creepypastas &&
+          creepypastas
+            .filter(creepypasta =>
+              creepypasta.title.trim().toLowerCase().includes(searchValue)
```
Comment (ahohnsen) [inline src/pages/CreepypastaPage.js:24]: Cause you are using these lines below again to check for the error, you could save the filtered results probably in a variable and use the variable to map over the results and to check the length to display the error message.

File: src/pages/CreepypastaPage.js [hunk 5]
```diff
@@ -1,27 +1,45 @@
...
+                title={creepypasta.title}
+                image={creepypasta.image}
+                wordcount={creepypasta.wordcount}
+                onBookmarkClick={() => handleBookmarkClick(creepypasta.id)}
+                isBookmarked={creepypasta.isBookmarked}
+              />
+            ))}
+        {creepypastas &&
+          creepypastas.filter(creepypasta =>
+            creepypasta.title.trim().toLowerCase().includes(searchValue)
```
Comment (ahohnsen) [inline src/pages/CreepypastaPage.js:39]: See above

**Review by ahohnsen — APPROVED** [review]
I like your code, it looks well structured to me. I only had a question (see my comment) and minor suggestions to maybe save you some lines of code :)
Well done!

File: src/components/Searchbar.js (hunk 2 above)
Comment (celinevo) [reply src/components/Searchbar.js:7]: It is for accessibility. I read it somewhere and just put it in, but I am not entirely sure if it is really necessary. :D

Comment (ahohnsen) [reply src/components/Searchbar.js:7]: Ah ok, good to know. I guess it is similar to the sr-only class we used once, right? Will have a look into that, thx.

**Pull request merged**